app/Mail/MailgunEmail.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\Mail;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Mail\Mailable;
+use Illuminate\Mail\Mailables\Content;
+use Illuminate\Mail\Mailables\Envelope;
+use Illuminate\Queue\SerializesModels;
+
+class MailgunEmail extends Mailable
+{
+    use Queueable, SerializesModels;
+
+    /**
+     * Create a new message instance.
+     */
+    public $data;
+
+    public function __construct($data)
+    {
+        $this->data=$data;
+    }
+
+    public function build(){
+         return $this->subject('please check')->view('contactus')->with('data',$this->data);
```

Comment:
you are supposed to use either build or envelope and content also i couldn't find the view you use to send the email with user data entered in the form